components/Templates/includes/auto-template/Pods_Templates_Auto_Template_Front_End.php

```diff
@@ -49,25 +49,12 @@ public function __construct() {
 	 * @since 2.6.6
 	 */
 	public function hook_content() {
-		/**
-		 * Allows plugin to append/replace the_excerpt.
-		 *
-		 * Default is false, set to true to enable.
-		 */
-		if ( ! defined( 'PFAT_USE_ON_EXCERPT' ) ) {
-			define( 'PFAT_USE_ON_EXCERPT', false );
-		}
-
 		$possible_pods = $this->auto_pods();
 
 		foreach ( $possible_pods as $pod_name => $pod ) {
 			$filter = $this->get_pod_filter( $pod_name );
 
 			$this->filtered_content[ $filter ] = 10.5;
```

Comment:
Since the changes on getting the single or archive filter name it's possible that no filter is returned, this should be checked.
The alternative is to set a default like before this PR (single).

Reply:
Updated